src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    return (price -= 50);
```

Comment:
Instead of modifying and returning the 'price' variable in one statement, you can simply return 'price - 50'. This makes the intention of your code clearer and avoids using the assignment operator in a return statement, which can be confusing.